recipes/aiofile/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "aiofile" %}
+{% set version = "3.7.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiofile-{{ version }}.tar.gz
+  sha256: 0aefa1d91d000d3a20a515d153db2ebf713076c7c94edf2fca85d3d83316abc5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - caio ~=0.9.0
```

Comment:
I think this way of specifying constrained is not fully supported now (and in fact we patch it for backwards compatibility). Can you elaborate on why this is needed?

Reply:
It's from upstream: https://github.com/mosquito/aiofile/blob/9915207c8564c4b49d57ce16003c385a42aa935b/setup.py#L61

But yeah you're right, completely forgot that it's not supported. Have translated it to conda specs.

Reply:
ICYI https://github.com/conda-incubator/grayskull/issues/270